server/aws-lsp-codewhisperer/src/language-server/codeWhispererSecurityScanServer.ts

```diff
@@ -4,6 +4,7 @@ import {
     ExecuteCommandParams,
     Server,
 } from '@aws/language-server-runtimes/server-interface'
+import { performance } from 'perf_hooks'
```

Comment:
Seems to be unused?

Reply:
Actually it is the opposite, it is used but was not imported previously